### PR TITLE
Update schemas for ASDF standard 1.6.0

### PR DESCRIPTION
**Pull request by braingram**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -2,22 +2,22 @@
...
     'Programming Language :: Python :: 3.9',
     'Programming Language :: Python :: 3.10',
+    'Programming Language :: Python :: 3.11',
+    'Programming Language :: Python :: 3.12',
     'Development Status :: 5 - Production/Stable',
 ]
 dependencies = [
-    'asdf-standard >= 1.0.1',
+    #'asdf-standard >= 1.0.1',
+    'asdf-standard @ git+https://github.com/braingram/asdf-standard.git@split_core',
```
Comment (braingram) [inline pyproject.toml:19]: To be removed prior to merge and after https://github.com/asdf-format/asdf-standard/pull/422 is merged and released.

File: pyproject.toml [hunk 2]
```diff
@@ -43,9 +43,9 @@ docs = [
 ]
 test = [
     'pytest >= 4.6.0',
-    'pytest-openfiles >= 0.5.0',
     'asdf >= 2.8.0',
-    'asdf-astropy',
+    #'asdf-astropy',
+    'asdf-astropy @ git+https://github.com/braingram/asdf-astropy.git@split_core',
```
Comment (braingram) [inline pyproject.toml:48]: To be removed prior to merge. This is at the moment required due to the transform/coordinate updates referenced in this PR (for example, one of the included examples uses the `remap_axis-1.4.0` tag which is added to the schema package in https://github.com/asdf-format/asdf-transform-schemas/pull/110 and supported by asdf-astropy in https://github.com/astropy/asdf-astropy/pull/219).

**Review by nden — APPROVED** [review]
LGTM

**Comment by braingram** [comment]
I missed this until now but the `$ref` here (on main):
https://github.com/asdf-format/asdf-wcs-schemas/blob/4d01f273cb0336405472fabb3f4aa1245aafbd21/resources/schemas/stsci.edu/gwcs/frame-1.0.0.yaml#L62
means that this package should depend on `asdf-coordinates-schemas` (it already depends on `asdf-transform-schemas`).

I added `asdf-coordinates-schemas` as a dependency and I think that slightly changes the plans for release to be:
- merge and release https://github.com/asdf-format/asdf-transform-schemas/pull/110
- merge and release https://github.com/asdf-format/asdf-coordinates-schemas/pull/55
- update this PR to add lower pins for the above packages
- merge and release this PR
- update asdf-astropy to be aware of the above changes
- continue... https://github.com/asdf-format/asdf/pull/1744

**Pull request merged**